XamarinCommunityToolkit/Converters/MultiConverterParameter.shared.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Represents a parameter to be used in the MultiConverter.
+    /// </summary>
+    public class MultiConverterParameter : BindableObject
+    {
+        public Type ConverterType { get; set; }
```

Comment:
probably we should static BindableProperty ConverterTypeProperty & ValueProperty if we'd like to try to make them bindable

Reply:
But I am not sure we can make them bindable in deed :(